crates/sui-framework/packages/deepbook/tests/clob_tests.move

```diff
@@ -840,6 +852,327 @@ module deepbook::clob_test {
         end(test)
     }
 
+    fun test_absorb_all_liquidity_bid_side_with_customized_tick_(test: Scenario): TransactionEffects {
+        let (alice, bob) = people();
+        // setup pool and custodian
+        let owner: address = @0xF;
+        next_tx(&mut test, owner);
+        {
+            clob::setup_test_with_tick_lot(5000000, 2500000, 1_00_000_000, 10, &mut test, owner);
+            mint_account_cap_transfer(alice, test::ctx(&mut test));
+        };
+
+        next_tx(&mut test, alice);
+        {
+            let pool = test::take_shared<Pool<SUI, USD>>(&mut test);
+            let account_cap = test::take_from_sender<AccountCap>(&test);
+            let account_cap_user = object::id(&account_cap);
+            let (base_custodian, quote_custodian) = clob::borrow_mut_custodian(&mut pool);
+            custodian::deposit(base_custodian, mint_for_testing<SUI>(1000, ctx(&mut test)), account_cap_user);
+            custodian::deposit(quote_custodian, mint_for_testing<USD>(10000, ctx(&mut test)), account_cap_user);
+            test::return_shared(pool);
+            test::return_to_sender<AccountCap>(&test, account_cap);
+        };
+
+        // alice palce series limit order
```

Comment:
```suggestion
        // alice place series limit order
```

Reply:
we did code clean in test code just now.